ui/runs/forms/custom_fields.py

```diff
@@ -87,6 +88,69 @@ def __init__(self, choices: Enum | list, initial=None, *args, **kwargs):
     def clean(self, value: list[str] | None):
         return [el for el in value if el != "hidden"] if value else None
 
+class CustomCheckboxMultipleChoiceField(MultipleChoiceField):
+    def __init__(self, choices: Enum | list, colors: Enum | list, initial=None, *args, **kwargs):
+        if isinstance(choices, list):
+            super().__init__(choices=choices, initial=initial, *args, **kwargs)
+        else:
+            super().__init__(
+                choices=[(el.value, el.value) for el in choices],
```

Comment:
Das "el in choices" ist nicht so clean, benenne das lieber in sowas wie choice in choices um. Gerne dann auch für den Rest der Datei 👍